Author: Daniel Hahler <ubuntu@thequod.de>
Description: Better help in case the sysprof-module is not loaded. (LP: #30160)
--- a/sysprof.c
+++ b/sysprof.c
@@ -560,7 +560,14 @@
 		       "\n"
 		       "       modprobe sysprof-module\n"
 		       "\n"
-		       "as root.");
+		       "as root.\n"
+		       "\n"
+		       "You may need to build and install this module first:\n"
+		       "  1. Install the package sysprof-module-source\n"
+		       "  2. Execute \"m-a a-i sysprof-module\"\n"
+		       "  3. Load the module as described above\n"
+		       "  4. Restart sysprof\n"
+		       "See also /usr/share/doc/sysprof-module-source/README.Debian" );
 		
 		update_sensitivity (app);
 		return;
